src/collectors/mesos/mesos.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+
+"""
+
+Collects metrics from a mesos instance (master or slave).
+
+#### Example Configuration
+
+Typically metrics for the master are exposed on port 5050,
+while metrics for the slaves are exposed on port 5051.
+
+```
+    host = localhost
+    port = 5050
+```
+"""
+
+import diamond.collector
+import json
+import urllib2
+
+
+class MesosCollector(diamond.collector.Collector):
+
+    METRICS_PATH = "metrics/snapshot"
+
+    def get_default_config_help(self):
+        config_help = super(MesosCollector,
+                            self).get_default_config_help()
+        config_help.update({
+            'host': 'Hostname',
+            'port': 'Port',
+            'path': 'Metrics Path'
```

Comment:
I don't think we should be overriding this here and providing a hard, absolute path value for all machines. Seems undesirable. What was your intent with this? From my understanding, this is going to lump everything under one toplevel key, `mesos` and we'd have no way to differentiate between machines.